### PR TITLE
feat(textlint-fixer-formatter): add getFormatterList function

### PR DESCRIPTION
**Pull request by 0x6b**
Hi, could you review my preliminary `getFormatterList` implementation for `textlint-fixer-formatter`(#399)?

### CODE REVIEW

File: packages/textlint-fixer-formatter/test/textlint-fixer-formatter-test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// LICENSE : MIT
```
Comment (azu) [inline packages/textlint-fixer-formatter/test/textlint-fixer-formatter-test.js]: Can you convert it to TypeScript?

It is good that adding new test file as TypeScript.

Comment (0x6b) [reply packages/textlint-fixer-formatter/test/textlint-fixer-formatter-test.js]: It makes sense. Thanks. I also found that only `assert` is not meaningful. Trying to fix.

**Comment by azu** [comment]
@0x6b Can you forget to apply `prettier` formatter?

    # project root
    yarn run prettier

https://github.com/textlint/textlint/blob/daeafe18bf19b7b206d1d963b9195aeb4c8a8ea8/package.json#L30
textlint install `precommit` hook when `yarn install` at first time.
 So, we can apply prettier automatically, but it was not applied... mystery.

**Comment by 0x6b** [comment]
Mysterious. Maybe husky :dog: haven't work as expected recently. I will check my setup but, in the meantime, apply prettier manually

**Comment by azu** [comment]
Thanks for fixing!

**Comment by 0x6b** [comment]
Really appreciate your multiple reviews 😄 Please mark one of the tasks at #399 as done. Thank you!

**Comment by 0x6b** [comment]
> Mysterious. Maybe husky 🐶 haven't work as expected recently. I will check my setup but, in the meantime, apply prettier manually

I set up my own hooks at `~/.gitconfig#core.hookspath` and it prevents Husky from running (see https://github.com/typicode/husky/issues/81#issuecomment-282540588). Will open an issue for updating [CONTRIBUTING.md](https://github.com/textlint/textlint/blob/master/.github/CONTRIBUTING.md) about this.

**Pull request merged**